ui/src/test/java/de/ipb_halle/lbac/material/common/bean/MaterialEditPremissionTest.java

```diff
@@ -0,0 +1,226 @@
+        userBean.setCurrentAccount(publicUser);
+        materialBean.setUserBean(userBean);
+    }
+
+    @Test
+    public void test001_isDetailInformationEditable() {
+        //Create material with public user as owner
+        Structure s = createStructure(GlobalAdmissionContext.getPublicReadACL(), publicUser);
+        materialBean.getMaterialEditState().setMaterialToEdit(s);
+        materialBean.getMaterialEditState().setMaterialBeforeEdit(s);
+
+        //In creation mode always returns true
+        Assert.assertTrue(permissionBean.isDetailInformationEditable(COMMON_INFORMATION.toString()));
+
+        //In EDIT mode returns true because current user is owner
+        materialBean.setMode(MaterialBean.Mode.EDIT);
+        Assert.assertTrue(permissionBean.isDetailInformationEditable(COMMON_INFORMATION.toString()));
+
+        //In EDIT mode returns true because current user (admin) is in public ACL
+        userBean.setCurrentAccount(adminUser);
+        Assert.assertTrue(permissionBean.isDetailInformationEditable(COMMON_INFORMATION.toString()));
+
+        //In EDIT mode returns false because current user(public) is not in adminOnlyAcl
+        s.setACList(context.getAdminOnlyACL());
+        s.setOwner(adminUser);
+        userBean.setCurrentAccount(publicUser);
+        Assert.assertFalse(permissionBean.isDetailInformationEditable(COMMON_INFORMATION.toString()));
+
+        //In HISTORY mode no edit is allowed
+        materialBean.setMode(MaterialBean.Mode.HISTORY);
+        userBean.setCurrentAccount(adminUser);
+        Assert.assertFalse(permissionBean.isDetailInformationEditable(COMMON_INFORMATION.toString()));
+    }
+
+    @Test
+    public void test002_isDetailPanelVisible() {
+        //Create material with public user as owner
+        materialBean.setMode(MaterialBean.Mode.CREATE);
+
+        //No materialType set
+        Assert.assertFalse(permissionBean.isDetailPanelVisible(COMMON_INFORMATION.toString()));
+
+        //Structure has the detail information and in creationmode is always visible
+        materialBean.setCurrentMaterialType(MaterialType.STRUCTURE);
+        Assert.assertTrue(permissionBean.isDetailPanelVisible(COMMON_INFORMATION.toString()));
+
+        //Structure has not the taxonomy detailinformation
+        materialBean.setCurrentMaterialType(MaterialType.STRUCTURE);
+        Assert.assertFalse(permissionBean.isDetailPanelVisible(MaterialDetailType.TAXONOMY.toString()));
+
+        //In EDIT mode current user (public) has access to common information
+        userBean.setCurrentAccount(publicUser);
+        materialBean.setMode(MaterialBean.Mode.EDIT);
+        Structure s = createStructure(GlobalAdmissionContext.getPublicReadACL(), adminUser);
+        materialBean.getMaterialEditState().setMaterialToEdit(s);
+        materialBean.getMaterialEditState().setMaterialBeforeEdit(s);
+        Assert.assertTrue(permissionBean.isDetailPanelVisible(COMMON_INFORMATION.toString()));
+
```

Comment:
Missing tests:
- I'm owner and have no read perms
- I'm not owner and have no read perms
- extend test for Mode.HISTORY